### PR TITLE
tinc-1.1

### PR DESCRIPTION
**Pull request by awiouy**

### CODE REVIEW

**Comment by andlauer** [comment]
I agree with the changes

File: addons/networking/tinc/package.mk [hunk 1]
```diff
@@ -1,53 +1,48 @@
...
-# along with OpenELEC; if not, write to the Free Software Foundation,
-# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
-#####################################################################
-
+# along with OpenELEC. If not, see <http://www.gnu.org/licenses/>.
+################################################################################
 PKG_NAME="tinc"
-PKG_VERSION="1.0.24"
+PKG_VERSION="1.1pre11"
 PKG_REV="1"
```
Comment (stefansaraev) [inline addons/networking/tinc/package.mk]: please bump PKG_REV to 2 and update changelog accordingly

File: addons/networking/tinc/changelog.txt [hunk 2]
```diff
@@ -1,17 +1,2 @@
...
-
-4.1.3
-- Bumped tinc to 1.0.24
-
-4.1.2
-- Changed configuration directory
-
-4.1.1
-- Initial package for tinc-1.0.23
+- Initial package for tinc-1.1pre11
```
Comment (stefansaraev) [inline addons/networking/tinc/changelog.txt]: 4.3.2

File: addons/networking/tinc/changelog.txt [hunk 3]
```diff
@@ -1,17 +1,2 @@
 4.3.1
-- update to kodi
```
Comment (stefansaraev) [inline addons/networking/tinc/changelog.txt]: well we never remove old changelog entries, for no good reason :)

**Comment by awiouy** [comment]
Thank you for the feedback.
Done!

**Comment by stefansaraev** [comment]
could you please squash into one commit ? thanks.

**Pull request closed without merging**